e2e/src/utils/utils.js

```diff
@@ -427,3 +427,7 @@ export async function fundWallet(senderPrivateKey, recipientAddress, stableToken
     .sendAndWaitForReceipt({ from: senderAddress })
   console.log('Funding TX receipt', receipt)
 }
+
+export const createCommentText = () => {
+  return `${new Date().getTime()}-${parseInt(Math.random() * 100_000)}`
```

Comment:
Is there any benefit to this being a dynamic string? Seems unlikely, but it could make for flakier tests.

Reply:
I'd consider making it just a static comment.

Reply:
static comment would mean parallel e2e runs could step on each other since they all use the same wallet. I don't think the randomness here should case any test flake. More context here: https://github.com/valora-inc/wallet/pull/4845#discussion_r1478848911

Reply:
ah got it, thanks!